demo/index.html

```diff
@@ -50,14 +50,14 @@
       <paper-button raised onclick="modalAlert.toggle()">modal alert</paper-button>
       <simple-dialog id="modalAlert" modal role="alertdialog">
         <h2>Alert</h2>
-        <paper-dropdown-menu label="Draft to discard">
-          <paper-listbox class="dropdown-content">
+        <paper-dropdown-menu-light label="Draft to discard">
+          <paper-listbox class="dropdown-content" slot="dropdown-content">
```

Comment:
Could you also leave a note here about why this needs both `class="dropdown-content"` and `slot="dropdown-content"`?